.bumpversion.cfg

```diff
@@ -1,3 +1,3 @@
 [bumpversion]
-tag = True
-tag_name = "rc/{new_version}"
\ No newline at end of file
+tag_name = "rc/v{new_version}"
+current_version = 1.1.8
```

Comment:
is `current_version = 1.1.8` automatically updated every time we bump version?

Reply:
yes, bumpversion automatically adds/updates this field